Fixed unclarity about free users consuming content (#3907)
MicrosoftDocs/powerbi-docs: powerbi-docs/consumer/end-user-features.md

[!INCLUDE[consumer-appliesto-ynnn](../includes/consumer-appliesto-ynnn.md)]

> As a Power BI service customer with a Free license, you explore reports and dashboards in order to make business decisions. Those reports and dashboards are created by *designers* who have Power BI Pro or Premium Per User (PPU) licenses and must be put in a workspace backed by a premium capacity in order to benefit from [unlimited content sharing](https://docs.microsoft.com/en-us/power-bi/enterprise/service-premium-what-is?WT.mc_id=DP-MVP-5003435#unlimited-content-sharing) as part of Premium. In case user with a Pro or PPU license share content in a regular workspace, you need an identical license as the *_designers_* have. Pro and PPU users have the ability to share content with their colleagues and to control what their colleagues can and can't do with that content. Sometimes, designers share content by sending you links, and sometimes the content automatically installs and appears in Power BI under **Apps** or **Shared with me**.

Reviewer: ```suggestion
As a Power BI service customer with a Free license, you explore reports and dashboards in order to make business decisions. Those reports and dashboards are created by *designers* who have Power BI Pro or Premium Per User (PPU) licenses. The designers need to share content to a workspace backed by a Premium capacity, to benefit from [unlimited content sharing](../enterprise/service-premium-what-is.md#unlimited-content-sharing) as part of Premium. If a designer with a Pro or PPU license shares content in a regular workspace, you need the same license that the *designers* have. Pro and PPU users have the ability to share content with their colleagues and to control what their colleagues can and can't do with that content. Sometimes, designers share content by sending you links, and sometimes the content automatically installs and appears in Power BI under **Apps** or **Shared with me**.
```
